typescript/orval.config.ts

```diff
@@ -9,12 +9,7 @@ export default defineConfig({
       clean: true,
       target: "./generated",
       mode: "tags-split",
-      mock: {
```

Comment:
orval was incorrectly generating mock function types and preventing the build from working. we dont use msw mock files so we update the orval config to exclude them